altair_data_server/_altair_server.py

```diff
@@ -42,7 +45,16 @@ def __call__(self, data, fmt='json'):
                 extension=fmt,
                 headers={"Access-Control-Allow-Origin": "*"}
                 )
-        return {'url': self._resources[resource_id].url}
+
+        url = self._resources[resource_id].url
+
+        # proxy via jupyter-server-proxy only works under python 3
+        if six.PY3:
+            url_parts = urlparse(self._resources[resource_id].url)
+            # vega defaults to <base>/files, redirect it to <base>/proxy/<port>/<file>
+            url = '../proxy/{}{}'.format(url_parts.port, url_parts.path)
```

Comment:
If I understand correctly, this makes altair data server *only* work on remove jupyer proxy, correct? I think this line would break local use-cases that currently work correctly.

Reply:
We require the jupyte-server-proxy (which gets activated just by installing it) to be installed now under Python 3, so this works even locally (even though it obviously doesn't add any feature in that case). I've tested it locally and it works.